src/main/scala/is/hail/variant/VariantSampleMatrix.scala

```diff
@@ -1055,10 +1020,11 @@ class VariantSampleMatrix[T](val hc: HailContext, val metadata: VariantMetadata,
     val localSampleAnnotationsBc = sampleAnnotationsBc
 
     rdd
-      .flatMap { case (v, (va, gs)) =>
-        localSampleIdsBc.value.lazyMapWith2[Annotation, T, U](localSampleAnnotationsBc.value, gs, {
-          case (s, sa, g) => f(v, va, s, sa, g)
-        })
+      .flatMap {
+        case (v, (va, gs)) =>
```

Comment:
Formatting change.

I think I figured out how this happens.  If you have some extreme typo and reformat the file, the reformatter makes all sorts of crazy changes.  If you fix the typo locally without undoing the reformat (e.g., just reformatting again), you pick up changes like this.